pkgs/development/libraries/libcifpp/default.nix

```diff
@@ -3,30 +3,21 @@
 , boost
 , cmake
 , fetchFromGitHub
-, fetchpatch
 , eigen
 , zlib
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "libcifpp";
-  version = "5.1.0.1";
+  version = "5.1.2";
 
   src = fetchFromGitHub {
     owner = "PDB-REDO";
     repo = "libcifpp";
     rev = "refs/tags/v${finalAttrs.version}";
-    hash = "sha256-fAH7FIgJuitPUoacLnLs8uf9di5iM0c/2WHZqVjJOUE=";
+    hash = "sha256-wx5D0kNKetgc/8LFAgNxTAwni+lJb2rajsxh0AASpeY=";
   };
 
-  patches = [
```

Comment:
note that this removes the patch from the version 5.0.8 that is used for dssp. If that's fine then the PR looks good to me.

Reply:
Thanks for the review.
There is no problem removing this patch.
Thank you for checking dssp as well.